debugging/book-library/index.html

```diff
@@ -73,7 +74,7 @@ <h1>Library</h1>
     <table class="table" id="display">
       <thead class="thead-dark">
         <tr>
-          <th>Title</th>
+          <th id="">Title</th>
```

Comment:
Why do you need empty id here?

Reply:
The id attribute was added by mistake, So I have removed